src/classes/particle.ts

```diff
@@ -366,48 +384,48 @@ export class Particle {
         }
     }
 
-    public bubble() {
+    public bubble(): void {
         this.bubbler.bubble();
     }
 
-    public repulse() {
+    public repulse(): void {
         this.repulser.repulse();
     }
 
     /* ---------- tsParticles functions - particles interaction ------------ */
-    public link(p2: Particle) {
+    public link(p2: Particle): void {
         this.updater.link(p2);
     }
 
-    public attract(p2: Particle) {
+    public attract(p2: Particle): void {
         this.updater.attract(p2);
     }
 
-    public bounce(p2: Particle) {
+    public bounce(p2: Particle): void {
         this.updater.bounce(p2);
     }
 
-    public move(delta: number) {
+    public move(delta: number): void {
         this.updater.move(delta);
     }
 
-    public moveParallax() {
+    public moveParallax(): void {
         this.updater.moveParallax();
     }
 
-    public updateOpacity() {
+    public updateOpacity(): void {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after constructors.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)